features/step_definitions/framework_application_steps.rb

```diff
@@ -39,14 +39,14 @@
     rescue Capybara::ElementNotFound => e
       answer_all_dos_lot_questions "Edit"
       answer_all_service_questions "Add"
-      find_elements_by_xpath("//input[@value='Mark as complete']")[0].click
+      find_elements_by_xpath("//button[contains(normalize-space(text()), 'Mark as complete')]")[0].click
```

Comment:
I think this can be simplified using the `click_button` method

```suggestion
      click_button("Mark as complete")
```

Reply:
I tried doing something like that before and ended up getting redirected to the GOV.UK home page... does this pass for you when you run locally?

Reply:
lolwat

Reply:
Yes it worked for me locally